ui/lib/kv/addon/components/page/list.js

```diff
@@ -87,6 +87,7 @@ export default class KvListPageComponent extends Component {
     evt.preventDefault();
     pathIsDirectory(this.secretPath)
       ? this.router.transitionTo('vault.cluster.secrets.backend.kv.list-directory', this.secretPath)
-      : this.router.transitionTo('vault.cluster.secrets.backend.kv.secret.index', this.secretPath);
```

Comment:
I reverted this because this test was failing and I felt like it made more sense to navigate straight to details in this instance (instead of checking whether the secret is real or not from the overview route)
https://github.com/hashicorp/vault/blob/7c2050a68f656cc1441c00b0b1dae9c5f0281dfe/ui/tests/acceptance/secrets/backend/kv/kv-v2-workflow-edge-cases-test.js#L175-L193